gradle/libs.versions.toml

```diff
@@ -125,6 +125,7 @@ generativeai = { group = "com.google.ai.client.generativeai", name = "generative
 datastore = { group = "androidx.datastore", name = "datastore-preferences", version.ref = "datastore"}
 androidx-lifecycle-viewmodel-navigation3 = { module = "androidx.lifecycle:lifecycle-viewmodel-navigation3", version.ref = "lifecycleViewmodel" }
 androidx-lifecycle-viewmodel-ktx = { group = "androidx.lifecycle", name = "lifecycle-viewmodel-ktx", version.ref = "lifecycleViewmodel" }
+androidx-navigation3-adaptive = { module = "androidx.compose.material3.adaptive:adaptive-navigation3", version.ref = "navigation3" }
```

Comment:
Since this is under the `androidx.compose.material3.adaptive` group, I'd probably have the library name be `adaptive-navigation3`